cmd/podman/checkpoint.go

```diff
@@ -22,6 +23,11 @@ var (
 			Name:  "keep, k",
 			Usage: "keep all temporary checkpoint files",
 		},
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "checkpoint all running containers",
+		},
+		LatestFlag,
```

Comment:
Not for this PR, but if we're going to define LatestFlag in common.go, should we do the same for the All flag?

Reply:
I actually thought about it, but the verb ('running' in this case), would be different, so it has to be a function. As the definition of the flags happens outside of a function, it would also need to be moved inside the function, I think, and it sounded overly complicated in this case.

Reply:
Oh good point on the running bit, that verbiage is probably different amongst the different podman commands.